doc/changes/devel/12955.newfeature.rst

```diff
@@ -0,0 +1 @@
+Add convenience :attr:`mne.io.BaseRaw.duration` property to centralize duration calculation for the textual (``__repr__``) and html (``_repr_html_``, used by e.g. Jupyter) display of :class:`mne.io.BaseRaw` instances, by :newcontrib:`Leonardo Rochael Almeida`.
```

Comment:
For historical and discoverability reasons we document these things for Raw not BaseRaw
```suggestion
Add convenience :attr:`mne.io.Raw.duration` property to centralize duration calculation for the textual (``__repr__``) and html (``_repr_html_``, used by e.g. Jupyter) display of :class:`mne.io.BaseRaw` instances, by :newcontrib:`Leonardo Rochael Almeida`.
```
Will push a commit, which will also get all CIs to run